vignettes/Data_processing.Rmd

```diff
@@ -1496,6 +1477,43 @@ fullPlot
 
 **Figure 4:** This figure illustrates the results from the Level 5 analyses in the <font face="CMTT10"> tcpl </font> pipeline package including the best model fit and subsequent point-of-departure (POD) estimates. The light-blue shaded region represents the estimated efficacy cutoff ($\mathit{coff}$). Each of the concentration-response models fit in Level 4 are included in the plot, where the blue curve indicates the best model fit for the observed data (white circles) and the rest are depicted by the gray curves.  The horizontal lines show the activity responses from which potency estimates of interest are defined, and the vertical lines show the corresponding POD estimates.  The black point shows the AC~5~ (concentration producing $5 \%$ of the maximal response), the purple point shows the AC~10~ (concentration producing $10 \%$ of the maximal response), the yellow point shows the BMD (benchmark dose), the orange point shows the ACC (concentration producing a response at the efficacy cutoff), and the pink point shows the AC~50~ (concentration producing $50 \%$ of the maximal response).
 
+#### Fit Categories
+
+<center>![<font style="font-size:15px"><i>Fit Category Tree</i></font>](img/Fig5_mc5_fitc.png)</center>
+
+**Figure 5:** All concentration series enter the fit category ($\mathit{fitc}$) hierarchical tree and move through activity, efficacy,
```

Comment:
Lines 1485-1489: rephrase as "Figure 5: A hierarchical fit category (fitc) decision tree is used to bin each fit. Each fit falls into one leaf of the tree using the described logic with the final fitc indicated with gray boxes. Abbreviations are defined as: Conc = concentration; hitc = hit call; |top| = absolute value of the modeled curve top; coff = cutoff; logc(min) = minimum log10 concentration tested; logc(max) = maximum log10 concentration tested; AC50 = 50% activity concentration; AC95 = 95% activity concentration.

After curve fitting, all concentration series are assigned a fit category (fitc). Concentration series in the same category will have similar characteristics, and often look very similar. Categorizing all series enables faster quality control checking and easier identification of potential false results. Logic is based on relative activity, efficacy, and potency comparisons as shown in Figure 5. For continuity purposes, fitc numbering has been conserved from past tcpl versions."

Reply:
I just want to confirm, you want concentration abbreviated as "Conc"  and not "conc"?  Reference line: "Abbreviations are defined as: Conc = concentration;"

Reply:
It can be updated as conc, not Conc. Just copied what was there

Reply:
That makes sense. Thank you.